package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Social bookmarking service",
   "main": "src/backend/index.js",
   "scripts": {
-    "dev": "./node_modules/.bin/nodemon --exec 'babel-node ./src/backend/index.js'",
-    "start": "babel-node ./src/backend/index.js",
+    "dev": "./node_modules/.bin/nodemon --exec 'babel-node ./server.js'",
```

Comment:
Perhaps I couldn't explain myself better. The `server.js` file should exist under the `backend` directory. The trick is that the `server.js` should define the server (I mean the init function) and the index.js file should pass true after calling the `init` function.

Reply:
so should i  keep both the `index.js` and `server.js` in the same directory that is the backend folder? or i should keep the `index.js` in the root directory